libs/core/src/lib/wizard/wizard-next-step/wizard-next-step.component.ts

```diff
@@ -11,4 +11,14 @@ export class WizardNextStepComponent {
      */
     @Input()
     floating = false;
+
+    /** @hidden */
+    displayed = true;
+
+    constructor(private _elRef: ElementRef) {}
```

Comment:
please add `readonly`

Reply:
Child properties of this elRef are modified by wizard component